plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExBudget.hs

```diff
@@ -118,15 +120,18 @@ instance ExBudgetBuiltin fun () where
     exBudgetBuiltin _ = ()
 
 -- This works nicely because @m@ contains @term@.
-class (ExBudgetBuiltin fun exBudgetCat, ToExMemory term) =>
-            SpendBudget m fun exBudgetCat term | m -> fun exBudgetCat term where
+class (ExBudgetBuiltin fun exBudgetCat) =>
```

Comment:
What is the motivation behind this removal?

Reply:
I was just following up the chain of type variables that this lets me remove, and it let me remove that, one, so I guess it's not needed!

Reply:
Nvm, I saw your earlier comment about budgeting and ST